lib/Avatar/index.js

```diff
@@ -5,66 +5,55 @@ import cx from 'classnames';
 /**
  * @usage
  *
- * <Avatar initials="MR" color="green" url="img/url" onlyInitials isHighlighted offline />
+ * <Avatar name="Seymour Butts" color="green" url="img/url" offline />
```

Comment:
You been watching heavy weights? lol

Reply:
This was just a name used in the stories previously. I might change it lol